apidoc/NdefRecord.yml

```diff
@@ -10,12 +12,17 @@ description: |
     <Modules.Nfc.createNdefRecordExternal>, <Modules.Nfc.createNdefRecordMedia>, <Modules.Nfc.createNdefRecordSmartPoster>,
     <Modules.Nfc.createNdefRecordText>, <Modules.Nfc.createNdefRecordUnknown>, or <Modules.Nfc.createNdefRecordUri> method 
     variants to create an NDEF record.
+    
+    **iOS**:
+    The NFC-functionality is available on iOS 11 and later. It currently only allows
+    to read specific NDEF-tags - writing tags is not supported by iOS so far.
 
     See also:
     [NdefRecord](http://developer.android.com/reference/android/nfc/NdefRecord.html)
 extends: Titanium.Proxy
-since: "1.0.0"
-platforms: [android]
+since: {android: "1.0.0", iphone: "6.3.0", ipad: "6.3.0"}
+osver: {ios: {min: "11.0"}}
+platforms: [android, iphone, ipad]
```

Comment:
As I see at other places in document, only 'iphone' is supported. I think we should remove' ipad' from above as well.

Reply:
Good catch!